--- ./Makefile.org	Wed Aug 22 12:52:45 2001
+++ ./Makefile	Mon Nov  5 21:25:32 2001
@@ -143,13 +143,13 @@
 # we will use the first valid entry in the search path
 MAN_PATH := $(firstword $(MSP))
 ifeq (,$(MAN_PATH))
-  MAN_PATH := /usr/man
+  MAN_PATH := /usr/share/man
 endif
 MAN_DIR1=$(MAN_PATH)/$(MAN_TYPE1)
 MAN_DIR7=$(MAN_PATH)/$(MAN_TYPE7)
 
 # pick an appropriate BIN path
-BIN_DIR = /usr/sbin
+BIN_DIR = /sbin
 
 # look for SMP support in config.h
 SMP := $(shell $(CC) -E -dM $(KCFG_FILE) | grep CONFIG_SMP | awk '{ print $$3; }')
@@ -202,13 +202,13 @@
 	cd src && (make veryclean)
 
 install: all  $(MANFILE1).gz $(MANFILE7).gz 
-	/usr/bin/install -D -b -m 644 bin/ians.o $(MODULES_DIR)/ians.o
-	/sbin/depmod -a $(KVER)
-	@echo "module installed under $(MODULES_DIR)"
-	/usr/bin/install -b -m 744 bin/ianscfg	$(BIN_DIR)
-	/usr/bin/install -b -m 744 bin/iansd	$(BIN_DIR)
-	/usr/bin/install -D -m 644 $(MANFILE1).gz $(MAN_DIR1)/$(MANFILE1).gz
-	/usr/bin/install -D -m 644 $(MANFILE7).gz $(MAN_DIR7)/$(MANFILE7).gz
+	mkdir -p $(DESTDIR)$(MODULES_DIR) $(DESTDIR)$(BIN_DIR) $(DESTDIR)$(MAN_DIR)
+	/usr/bin/install -D -b -m 644 bin/ians.o $(DESTDIR)$(MODULES_DIR)/ians.o
+	@echo "module installed under $(DESTDIR)$(MODULES_DIR)"
+	/usr/bin/install -b -m 744 bin/ianscfg	$(DESTDIR)$(BIN_DIR)/ianscfg
+	/usr/bin/install -b -m 744 bin/iansd	$(DESTDIR)$(BIN_DIR)/iansd
+	/usr/bin/install -D -m 644 $(MANFILE1).gz $(DESTDIR)$(MAN_DIR1)/$(MANFILE1).gz
+	/usr/bin/install -D -m 644 $(MANFILE7).gz $(DESTDIR)$(MAN_DIR7)/$(MANFILE7).gz
 
 
 uninstall:
