--- ./src/Makefile.org  Fri Feb 22 23:12:36 2002
+++ ./src/Makefile      Fri Feb 22 23:14:13 2002
@@ -156,12 +156,12 @@
 # we will use the first valid entry in the search path
 MAN_PATH := $(firstword $(MSP))
 ifeq (,$(MAN_PATH))
-  MAN_PATH := /usr/man
+  MAN_PATH := /usr/share/man
 endif
 MAN_DIR=$(MAN_PATH)/$(MAN_TYPE)

 # pick an appropriate BIN install path
-BIN_DIR = /usr/sbin
+BIN_DIR = /sbin

 ###########################################################################
 # Driver files
--- ./src/Makefile.org	Fri Feb 22 22:45:52 2002
+++ ./src/Makefile	Fri Feb 22 23:05:44 2002
@@ -247,20 +247,19 @@
 	@echo "Compiling $< ..."
 	$(CC) $(CFLAGS) -c $< -o $@
 
-$(MAN_GZ): $(MANFILE)
+$(MAN_GZ): $(MANFILE)dd
 	gzip -c $< > $@
 
-install: all $(MANFILE).gz
-	install -D -m 444 $(CORE_DIR)/$(MODULE)     $(MOD_DIR)/$(MODULE)
-	install -D -m 544 $(CORE_DIR)/$(CORE_BIN1)  $(BIN_DIR)/$(CORE_BIN1)
-	install -D -m 544 $(CORE_DIR)/$(CORE_BIN2)  $(BIN_DIR)/$(CORE_BIN2)
-	install -D -m 444 $(MAN_GZ)                 $(MAN_DIR)/$(MAN_GZ)
+install: all $(MANFILE)
+	install -D -m 444 $(CORE_DIR)/$(MODULE)     $(DESTDIR)$(MOD_DIR)/$(MODULE)
+	install -D -m 544 $(CORE_DIR)/$(CORE_BIN1)  $(DESTDIR)$(BIN_DIR)/$(CORE_BIN1)
+	install -D -m 544 $(CORE_DIR)/$(CORE_BIN2)  $(DESTDIR)$(BIN_DIR)/$(CORE_BIN2)
+	install -D -m 444 $(MANFILE)                $(DESTDIR)$(MAN_DIR)/$(MANFILE)
 
-	depmod -a $(KVER)
-	@echo "module $(MODULE) installed under $(MOD_DIR)"
-	@echo "utility $(CORE_BIN1) installed under $(BIN_DIR)"
-	@echo "daemon $(CORE_BIN2) installed under $(BIN_DIR)"
-	@echo "man page $(MAN_GZ) installed under $(MAN_DIR)"
+	@echo "module $(MODULE) installed under $(DESTDIR)$(MOD_DIR)"
+	@echo "utility $(CORE_BIN1) installed under $(DESTDIR)$(BIN_DIR)"
+	@echo "daemon $(CORE_BIN2) installed under $(DESTDIR)$(BIN_DIR)"
+	@echo "man page $(MANFILE) installed under $(DESTDIR)$(MAN_DIR)"
 
 uninstall:
 	if [ -e $(MOD_DIR)/$(MODULE) ] ; then \
