--- ./src/Makefile.org	Tue Nov 20 11:26:49 2001
+++ ./src/Makefile	Sat Dec 15 15:22:48 2001
@@ -210,12 +210,12 @@
 # we will use the first valid entry in the search path
 MAN_PATH := $(firstword $(MSP))
 ifeq (,$(MAN_PATH))
-  MAN_PATH := /usr/man
+  MAN_PATH := /usr/share/man
 endif
 MAN_DIR=$(MAN_PATH)/$(MAN_TYPE)
 
 # pick an appropriate BIN install path
-BIN_DIR = /usr/sbin
+BIN_DIR = /sbin
 
 
 ###########################################################################
@@ -298,17 +298,17 @@
 	@echo "Compiling $< ..."
 	$(CC) $(CFLAGS) -c $< -o $@
 
-$(MANFILE).gz: $(MANFILE)
-	gzip -c $< > $@
+#$(MANFILE).gz: $(MANFILE)
+#	gzip -c $< > $@
 
-install: all $(MANFILE).gz
-	install -D -b -m 644 $(TARGET)		$(MOD_DIR)/ians.o
-	install -b -m 744 bin/$(ARCH)/ianscfg	$(BIN_DIR)
-	install -b -m 744 bin/$(ARCH)/iansd	$(BIN_DIR)
-	install -D -m 644 $(MANFILE).gz         $(MAN_DIR)/$(MANFILE).gz
+install: all $(MANFILE)
+	install -D -b -m 644 $(TARGET)		$(DESTDIR)$(MOD_DIR)/ians.o
+	install -D -b -m 744 bin/$(ARCH)/ianscfg	$(DESTDIR)$(BIN_DIR)/ianscfg
+	install -D -b -m 744 bin/$(ARCH)/iansd	$(DESTDIR)$(BIN_DIR)/iansd
+	install -D -m 644 $(MANFILE)		$(DESTDIR)$(MAN_DIR)/$(MANFILE)
 
-	depmod -a $(KVER)
-	@echo "module installed under $(MOD_DIR)"
+#	depmod -a $(KVER)
+	@echo "module installed under $(DESTDIR)$(MOD_DIR)"
 
 uninstall:
 	if [ -e $(MOD_DIR)/ians.o ] ; then \
