--- ./Makefile.org	Sun Jun 10 09:32:48 2001
+++ ./Makefile	Sun Sep 16 02:13:03 2001
@@ -128,8 +128,8 @@
 endif
 
 # Adjust the following paths to match the appropriate directories
-BIN_DIR = /usr/sbin
-MAN_DIR = /usr/man/man1
+BIN_DIR = /sbin
+MAN_DIR = /usr/share/man/man1
 
 # look for SMP support in config.h
 SMP := $(shell $(CC) -E -dM $(KCFG_FILE) | grep CONFIG_SMP | awk '{ print $$3; }')
@@ -173,15 +173,13 @@
 	cd src && (make veryclean)
 
 install:	all
-	mkdir -p $(MODULES_DIR)
-	test -e $(MAN_DIR) || ln -s /usr/share/man /usr/man
-	/usr/bin/install -b bin/ians.o	$(MODULES_DIR)
-	/usr/bin/install -b bin/ianscfg	$(BIN_DIR)
-	/usr/bin/install -b ianscfg.1		$(MAN_DIR)
+	mkdir -p $(DESTDIR)$(MODULES_DIR) $(DESTDIR)$(BIN_DIR) $(DESTDIR)$(MAN_DIR)
+	/usr/bin/install -b bin/ians.o	$(DESTDIR)$(MODULES_DIR)
+	/usr/bin/install -b bin/ianscfg	$(DESTDIR)$(BIN_DIR)
+	/usr/bin/install -b ianscfg.1 $(DESTDIR)$(MAN_DIR)
 
 	@echo "module installed under $(MODULES_DIR)"
 
-	/sbin/depmod -a $(KVER)
 
 uninstall:
 	/bin/rm -f $(BIN_DIR)/ianscfg
